<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HorizontalMapTargetRegionWidget</class>
 <widget class="QWidget" name="HorizontalMapTargetRegionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="targetGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>706</width>
       <height>350</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Target Region</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item alignment="Qt::AlignTop">
       <widget class="QRadioButton" name="mButtonCRDR">
        <property name="text">
         <string>Cross Range / Down Range</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mRangeGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="reportOptionsLayout">
         <item row="2" column="3">
          <widget class="UtQtLengthValueEdit" name="mDownRangeStepEdit">
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="UtQtLengthValueEdit" name="mDownRangeMaxEdit">
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Down Range</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Maximum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Step</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="UtQtLengthValueEdit" name="mCrossRangeStepEdit">
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="UtQtLengthValueEdit" name="mCrossRangeMaxEdit">
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="UtQtLengthValueEdit" name="mCrossRangeMinEdit">
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Cross Range</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Minimum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="UtQtLengthValueEdit" name="mDownRangeMinEdit">
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mButtonLatLon">
        <property name="text">
         <string>Latitude / Longitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mLatLonGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="reportOptionsLayout_2">
         <item row="0" column="3">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Step</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Latitude</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Maximum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Minimum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Longitude     </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="mRelativeCheckBox">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string>Relative Orientation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="UtQtLonPosLineEdit" name="mLonMaxEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="UtQtLatPosLineEdit" name="mLatMaxEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="UtQtLatPosLineEdit" name="mLatStepEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="UtQtLatPosLineEdit" name="mLatMinEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="UtQtLonPosLineEdit" name="mLonMinEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="UtQtLonPosLineEdit" name="mLonStepEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>134</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtLatPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLonPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mButtonCRDR</tabstop>
  <tabstop>mCrossRangeMinEdit</tabstop>
  <tabstop>mCrossRangeMaxEdit</tabstop>
  <tabstop>mCrossRangeStepEdit</tabstop>
  <tabstop>mDownRangeMinEdit</tabstop>
  <tabstop>mDownRangeMaxEdit</tabstop>
  <tabstop>mDownRangeStepEdit</tabstop>
  <tabstop>mButtonLatLon</tabstop>
  <tabstop>mLatMinEdit</tabstop>
  <tabstop>mLatMaxEdit</tabstop>
  <tabstop>mLatStepEdit</tabstop>
  <tabstop>mLonMinEdit</tabstop>
  <tabstop>mLonMaxEdit</tabstop>
  <tabstop>mLonStepEdit</tabstop>
  <tabstop>mRelativeCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
